pulp_container/tests/functional/conftest.py

```diff
@@ -29,6 +31,8 @@
     BearerTokenAuth,
 )
 
+from .utils import gen_repo, gen_container_remote
```

Comment:
Are these functions really worth importing?

Reply:
I think so. There's some things imported from `pulp_smash` into those functions, which could be later changed to some local solution without changing the tests itself.

Reply:
One of the goals is to not use anything from pulp-smash for exactly that reason. "It can be changed elsewhere, giving an unstable test experience."